app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,15 @@
 class PostsController < ApplicationController
-  before_action :authenticate_user!
+  # before_action :authenticate_user!
   load_and_authorize_resource
 
   def index
     @user = User.find(params[:user_id])
     @posts = Post.where(author_id: @user.id)
+
+    respond_to do |format|
+      format.html
```

Comment:
- [ ] When I use Postman to get the posts, I get html, kindly make sure when a user access the app as API it should return the data in a json format not html

Reply:
@Ismailco Thanks for your kind feedback but we did it because we wanted it to work for browser too, 
To receive a response as `json` in postman you should set the `Accept` header to `application/json`